packages/cdktf-cli/templates/csharp/Main.cs

```diff
@@ -5,13 +5,8 @@
 
 namespace MyCompany.MyApp
 {
-    class MyApp : TerraformStack
+    class MyApp : Program
```

Comment:
Should probably rename the file to `Program.cs`. Will need to update hooks and some tests to account for that change.